test/integration/testTwilioHelpers.js

```diff
@@ -0,0 +1,77 @@
+// Third-party dependencies
+const chai = require('chai')
+const expect = require('chai').expect
+const { afterEach, beforeEach, describe, it } = require('mocha')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+
+// In-house dependencies
+const helpers = require('../../lib/helpers')
+const twilioHelpers = require('../../lib/twilioHelpers')
+
+chai.use(sinonChai)
+
+const sandbox = sinon.createSandbox()
```

Comment:
Sinon is a mocking library. It allows us to check whether functions are called, how many times, with what parameters, and to define return values, if we want.

Reply:
Sounds good.